app/build.gradle

```diff
@@ -5,6 +5,9 @@ plugins {
     id 'com.google.dagger.hilt.android'
```

Comment:
The PR`s size is huge. Please do smaller requests next times

Reply:
My mistake with ktlint. I'll also try to figure out how to commit only the necessary changes.